src/utils/hints/namesHints.js

```diff
@@ -0,0 +1,14 @@
+export const namesHints = {
+  name: 'Unique identifier that may be configured to point to accounts, contracts, oracles, and NFTs.',
```

Comment:
As far as I know NFTs wrapping is still in progress so maybe I would remove it for now:

`Unique identifier that may be configured to point to accounts, contracts or oracles.`

Reply:
I guess, you are right. I took the knowledge from our landing page.
![Uploading image.png…]()

Fixed